group_project_v2/mixins.py

```diff
@@ -133,13 +133,28 @@ def _get_setting(self, setting, default):
         return result
 
 
-class AuthXBlockMixin(SettingsMixin, ProjectAPIXBlockMixin):
+class AuthXBlockMixin(SettingsMixin, ProjectAPIXBlockMixin, CourseAwareXBlockMixin):
 
-    DEFAULT_TA_ROLE = ('assistant',)
+    DEFAULT_TA_ROLE = ("assistant", )
 
     _ACCESS_DASHBOARD_ROLE_GROUPS_KEY = "access_dashboard_groups"
     _ACCESS_DASHBOARD_FOR_ALL_ORGS_GROUPS_KEY = "access_dashboard_for_all_orgs_groups"
-    _TA_ROLES_KEY = 'ta_roles'
+    _ACCESS_DASHBOARD_TA_ROLES = "access_dashboard_ta_groups"
+    _TA_ROLES_KEY = "ta_roles"
+
+    @property
+    def see_dashboard_ta_perms(self):
```

Comment:
@jbzdak please change them so it is lcear thy are the same thing, as you see fit.